interfaces/builtin/basedeclaration.go

```diff
@@ -50,25 +171,30 @@ slots:
     allow-installation:
       slot-snap-type:
         - core
+    deny-connection:
+      plug-attributes:
+        allow-sandbox: true
   camera:
     allow-installation:
       slot-snap-type:
         - core
     deny-auto-connection: true
   content:
-    allow-auto-connection:
-      plug-publisher-id:
-        - $SLOT_PUBLISHER_ID
     allow-installation:
       slot-snap-type:
         - app
         - gadget
+    allow-auto-connection:
+      plug-publisher-id:
+        - $SLOT_PUBLISHER_ID
   cups-control:
     allow-installation:
       slot-snap-type:
         - core
     deny-auto-connection: true
   docker:
+    allow-installation: false
+    deny-connection: true
     deny-auto-connection: true
   docker-support:
```

Comment:
I was thinking that denying installation and auto connection was enough for the super-privileged implicit slots. Is there something I am missing?